### PR TITLE
Change feature lists to individual sections

### PR DESCRIPTION
**Pull request by syeopite**
There's also some undraw illustration on this. I haven't discussed it in the matrix room yet but I think they look pretty nice. Feel free to remove it when merging.

new preview

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
       <h5 class="mt-0">Chat with us:</h5>
         <a href="https://matrix.to/#/#invidious:matrix.org"><img alt="Matrix" src="https://img.shields.io/matrix/invidious:matrix.org?label=Matrix&color=darkgreen"></a>
         <a href="https://web.libera.chat/?channel=#invidious"><img alt="Libera.chat (IRC)" src="https://img.shields.io/badge/IRC%20%28Libera.chat%29-%23invidious-darkgreen"></a>
+
+        <br/>
```
Comment (TheFrenchGhosty) [inline layouts/index.html:18]: Why here?

File: layouts/index.html [hunk 2]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
...
-            <li>Light/Dark themes
-            <li>Customizable homepage
-            <li>Subscriptions independant from Google
-            <li>Notifications for all subscribed channels
-            <li>Audio-only mode (with background play on mobile)
-            <li>Support for Reddit comments
-            <li>Available in many languages, thanks to our translators
-          </ul>
-        </p>
+        <img src="undraw_regain_focus_ecvj.svg" height=250/>
```
Comment (TheFrenchGhosty) [inline layouts/index.html:37]: I'm really not a fan of this illustration.

The "design aspiration"  illustration is better in my opinion.

File: layouts/index.html [hunk 3]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
...
+        <h2 class="h4">Humanely designed</h2>
```
Comment (TheFrenchGhosty) [inline layouts/index.html:38]: "Ethically designed" might be better.

File: layouts/index.html [hunk 4]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
...
+        <p> Invidious helps you regain your focus through ethical design! No more shall you conclusively check YouTube and waste your day away!</p>
```
Comment (TheFrenchGhosty) [inline layouts/index.html:39]: We should mention that users won't be affected by "bad" features like recommendations and other "addictive" features.

File: layouts/index.html [hunk 5]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
...
-            <li>Developer API
-            <li>Does not use official YouTube APIs
-            <li>No Contributor License Agreement (CLA)
-          </ul>
-        </p>
+        <img src="undraw_subscriptions_re_k7jj.svg" height=250/>
+        <h2 class="h4">Accounts</h2>
+        <p> Invidious allows you to keep track of channels and create playlists, all entirely independent from YouTube </p>
       </div>
+
```
Comment (TheFrenchGhosty) [inline layouts/index.html:46]: Why?

File: layouts/index.html [hunk 6]
```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
...
     </div>
+
```
Comment (TheFrenchGhosty) [inline layouts/index.html:48]: Why?

**Review by TheFrenchGhosty — CHANGES_REQUESTED** [review]

**Comment by TheFrenchGhosty** [comment]
The undraw illustration are definitely a great idea (can you add alt text to them) this definitely makes things better.

We should maybe follow the "default" Doks design and go back to the 6 "features" listed, that way we can mention more things.

Also: I don't see the Matrix button on your end but I see that it's still in the code, I'm guessing it's an issue on your end?

File: layouts/index.html (hunk 1 above)
Comment (syeopite) [reply layouts/index.html:18]: Imo, It's a bit too cramped without a buffer between the heading section and the feature overview.

With buffer:
w buffer

Without buffer:
no buffer

**Comment by syeopite** [comment]
PR now includes some slight color modifications to reduce the amount of accent-colors we have. Wdyt? I'm not sure if I'm a big fan of the colors I choose for the primary buttons though.

> Also: I don't see the Matrix button on your screenshot but I see that it's still in the code, I'm guessing it's an issue on your end?

Oops, I was testing some stuff and forgot to refresh before the screenshot. Should be fixed with the new one.

**Comment by SamantazFox** [comment]
Under which license are those pictures distributed?

**Comment by syeopite** [comment]
https://undraw.co/license

It's **very** permissive. We can pretty much do whatever we want as long as we don't pass off undraw artworks as our own.

File: layouts/index.html (hunk 1 above)
Comment (TheFrenchGhosty) [reply layouts/index.html]: True.

**Comment by TheFrenchGhosty** [comment]
Amazing! Great job!

**Review by TheFrenchGhosty — APPROVED** [review]

**Pull request merged**